### DISCUSSION TITLE
Tandoor LXC, wan access issue

### DISCUSSION THREAD

**Discussion by Narkowii** [tteck/Proxmox#2561] [Help Needed]
Hello, 

I have an issue accessing Tandoor LXC outside my network...
I use Nginx (in another LXC), but my config works with all of my VM and other LXC....

I have this, after logging in Tandoor :
<img width="449" alt="Tandoor HTTPS" src="https://github.com/tteck/Proxmox/assets/153772234/50857768-5bd2-497a-9299-303c1a460fdc">

**Comment by tteck** [comment]
https://github.com/TandoorRecipes/recipes/blob/develop/docs/system/configuration.md

**Reply by Narkowii** [reply]
Thanks, i can access now, but i don't see the any recipes....i search, perhaps something about the location of some directories....

**Reply by Narkowii** [reply]
I find it....in the nginx conf...

i change this ` proxy_set_header X-Forwarded-Proto $scheme;` by ` proxy_set_header X-Forwarded-Proto 'https';`
